contracts/JBERC20PaymentTerminal.sol

```diff
@@ -42,7 +42,9 @@ contract JBERC20PaymentTerminal is JBPaymentTerminal {
     address payable _to,
     uint256 _amount
   ) internal override {
-    IERC20(token).transferFrom(_from, _to, _amount);
+    _from == msg.sender ?
```

Comment:
shouldn't this be _from == address(this) ?

Reply:
yes, absolutely right!